index.html

```diff
@@ -158,18 +158,18 @@ <h2 class="contact-form-heading">Contact Me</h2>
         <form action="https://formspree.io/f/xbjwdrql" method="post" name="contactform" id="contact-Form" class="contact-form">
           <ul>
             <li class="form-field">
-              <small></small>
                <input type="text" maxlength="30" id="userName" class="username" placeholder="Enter your name">
+               <small></small>
             </li>
             <li class="form-field">
-              <small></small>
                <input type="email"  id="userEmail" class="username" placeholder="Enter your Email">
+               <small></small>
             </li>
             <li class="form-field">
-              <small></small>
               <textarea class="username" id="textareaMessage" cols="30" rows="10" maxlength="500" placeholder="Write your message"></textarea>
+              <small></small>
             </li>
-            <li>
+            <li class="form-field">
               <button type="submit" class="submit-btn">Get in Touch</button>
```

Comment:
- [ ] As suggested by the previous reviewer. Please kindly move the error message close to the submit button:

![image](https://user-images.githubusercontent.com/52934128/174336383-edd1e2b3-b72f-41d8-bc2d-37b7a4cac09a.png)